### PR TITLE
Introduces and auto-update mechanism

### LINKED ISSUE #2046: Auto Update Mechanism
## Abstract

It would be awesome to have an autoupdater for latest rotki releases.

## Motivation

Right now the users have to download the latest rotki release from github or through brew (1.11 not released at brew?).
This seems very complicated. An auto update mechanism could help to bring the latest features to the users without they have to do something.

## Specification

Could this help?
- https://www.electronjs.org/docs/tutorial/updates

### PR DESCRIPTION
**Pull request by kelsos**
Closes #2046

It should work with AppImage, nsis (windows installer) and dmg.

### CODE REVIEW

**Comment by LefterisJP** [comment]
So this only from v1.15.0 and not v1.14.2 I guess?

**Comment by kelsos** [comment]
Yes, this will work for any release after v1.15.0

File: frontend/app/src/locales/en.json [hunk 1]
```diff
@@ -893,6 +893,15 @@
...
     "version": "An Updated Version ({appVersion}) of Rotki is available"
   },
+  "update_popup": {
+    "action": {
+      "cancel": "Cancel",
+      "update": "Download",
+      "install": "Install"
+    },
+    "downloaded": "The updated version will be installed on the next restart. Would you like to restart the application now to apply the update?",
+    "messages": "A new version of rotki is available. You would like to download it?"
```
Comment (LefterisJP) [inline frontend/app/src/locales/en.json:903]:
```suggestion
    "messages": "A new version of rotki is available. Would you like to download it?"
```

**Review by LefterisJP — APPROVED** [review]
just small typo. Otherwise looks good.

**Pull request merged**